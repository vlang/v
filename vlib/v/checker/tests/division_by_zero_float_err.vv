fn main() {
	println(1.0 / 0.0)
}
