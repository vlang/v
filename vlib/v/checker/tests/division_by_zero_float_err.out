vlib/v/checker/tests/division_by_zero_float_err.vv:2:16: error: division by zero
    1 | fn main() {
    2 |     println(1.0 / 0.0)
      |                   ~~~
    3 | }
